include/NeoFOAM/core/primitives/scalar.hpp

```diff
@@ -10,4 +10,7 @@ typedef double scalar;
 #else
 typedef float scalar;
 #endif
+
+constexpr scalar ROOTVSMALL = 1e-18;
```

Comment:
```suggestion
constexpr scalar ROOTVSMALL = 10.0*std::numeric_limits<scalar>::epsilon();
```
I think this is bug waiting to happen? Smaller than eps?

Reply:
you will need `#include <limits>`

Reply:
this is a good point, but we need to have the exact same behavior as OF here. so for now we just copy the value.